editor/script_editor_debugger.cpp

```diff
@@ -642,7 +642,7 @@ void ScriptEditorDebugger::_parse_message(const String &p_msg, const Array &p_da
 							v /= 1024.0;
 						}
 						tt += " bytes";
-						vs = rtos(v) + " " + unit;
+						vs = rtos(Math::round(v * 100) * 0.01) + " " + unit;
```

Comment:
I'm pretty sure we have a built-in and more reliable way to do this. You coudl still get `32.00001 MB` with this method. There should be simple C float formatting instructions available.

Reply:
There is `static String::num()`, which takes in the number of decimals, and there is `static String::sprintf()`

Reply:
Well maybe I spoke too fast, I don't find a simpler way to do this. The alternative would be snprintf with two buffers...

Reply:
Ah of course in String yeah, I was looking at Math...

Reply:
so... what would be the best for it?

Reply:
`String::num(v, 2)`

Reply:
I couldn't find this beautiful method on GDscript.
why didn't this bind to gdscript?

Reply:
@volzhs there's [`String.pad_decimals()`](http://docs.godotengine.org/en/latest/classes/class_string.html#class-string-pad-decimals) in GDScript, but you can also use [format string](http://docs.godotengine.org/en/latest/learning/scripting/gdscript/gdscript_format_string.html) to achieve this.

Reply:
@vnen Thanks for the tip. I'm using format string. but I'm not aware of `pad_decimals`